File: data_structures/graphs/Python/graph_example.py
g = Graph()

g.add_vertex("a", 3, 3)
g.add_vertex("b", 7, 8)
g.add_vertex("c", 12, 0)
g.add_vertex("d", 6, 5)
g.add_vertex("e", 2, 9)
g.add_vertex("f", 1, 6)
g.add_vertex("g", 9, 2)
g.add_vertex("h", 13, 9)
g.add_vertex("i", 4, 7)
g.add_vertex("j", 11, 2)

g.add_edge("a", "i")
g.add_edge("a", "f")
g.add_edge("a", "g")
g.add_edge("e", "b")
g.add_edge("i", "e")
g.add_edge("g", "j")
g.add_edge("b", "d")
g.add_edge("b", "j")
g.add_edge("j", "c")
g.add_edge("j", "h")
g.add_edge("c", "h")

File: data_structures/graphs/JavaScript/graphExample.js
g = new Graph();

g.addVertex("a", 3, 3);
g.addVertex("b", 7, 8);
g.addVertex("c", 12, 1);
g.addVertex("d", 6, 5);
g.addVertex("e", 2, 9);
g.addVertex("f", 1, 6);
g.addVertex("g", 9, 2);
g.addVertex("h", 13, 9);
g.addVertex("i", 4, 7);
g.addVertex("j", 11, 2);

g.addEdge("a", "i");
g.addEdge("a", "f");
g.addEdge("a", "g");
g.addEdge("e", "b");
g.addEdge("i", "e");
g.addEdge("g", "j");
g.addEdge("b", "d");
g.addEdge("b", "j");
g.addEdge("j", "c");
g.addEdge("j", "h");
g.addEdge("c", "h");
